docs/dxp/7.x/en/installation-and-upgrades/installing-liferay-dxp-on-premises/installing-liferay-on-an-application-server/installing-liferay-on-jboss-eap.md

```diff
@@ -0,0 +1,350 @@
+            <resource-root path="mysql.jar" />
+            <resource-root path="portal-kernel.jar" />
+            <resource-root path="portlet.jar" />
+        </resources>
+        <dependencies>
+            <module name="javax.api" />
+            <module name="javax.mail.api" />
+            <module name="javax.servlet.api" />
+            <module name="javax.servlet.jsp.api" />
+            <module name="javax.transaction.api" />
+        </dependencies>
+    </module>
+    ```
+
+    For a different database, replace the MySQL `.jar` with the driver JAR for the database (e.g., HSQL, PostgreSQL, etc.).
+
+1. Create an `osgi` folder in the [Liferay Home](../../14-reference/01-liferay-home.md) folder. Extract the OSGi Dependencies ZIP file that you downloaded into the `[Liferay Home]/osgi` folder.
+
+    The `osgi` folder provides the necessary modules for DXP's OSGi runtime.
+
+**Checkpoint:**
+
+1. The dependencies files have been unzipped into the `$JBOSS_HOME/modules/com/liferay/portal/main` folder and a database jar.
+1. The `module.xml` has listed all JARs in the `<resource-root-path>` elements.
+1. The `osgi` dependencies have been unzipped into the `osgi` folder.
+
+### Running DXP on JBoss EAP in Standalone Mode vs. Domain Mode
+
+JBoss EAP can be launched in either *standalone* mode or *domain* mode. Domain mode allows multiple application server instances to be managed from a single control point. A collection of such application servers is known as a *domain*. For more information on standalone mode vs. domain mode, please refer to the section on this topic in the [JBoss EAP Product Documentation](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html/introduction_to_jboss_eap/overview_of_jboss_eap#operating_modes).
+
+DXP supports JBoss EAP when it runs in standalone mode but not when it runs in domain mode. DXP's hot-deploy does not work with a managed deployment, since JBoss manages the content of a managed deployment by copying files (exploded or non-exploded). This prevents JSP hooks and Ext plugins from working as intended. For example, JSP hooks don't work on JBoss EAP running in managed domain mode, since DXP's JSP override mechanism relies on the application server. Since JSP hooks and Ext plugins are deprecated, however, you may not be using them.
+
+The command line interface is recommended for domain mode deployments.
+
+| **Note:** This does not prevent DXP from running in a clustered environment on multiple JBoss servers. You can set up a cluster of DXP instances running on JBoss EAP servers running in standalone mode. Please refer to the [DXP clustering articles](https://help.liferay.com/hc/articles/360029123831-Liferay-DXP-Clustering) for more information.
+
+## Configuring JBoss
+
+Configuring JBoss to run DXP includes these things:
+
+* Setting environment variables
+* Setting properties and descriptors
+* Removing unnecessary configurations
```

Comment:
@justinchoi001 It would be good to break these into subsections and cross reference them.